sqldb/sqlc/queries/sequences.sql

```diff
@@ -0,0 +1,2 @@
+-- name: SelectAllSequences :many
```

Comment:
nit: maybe worth adding a `GetSequence` that selects an individual sequence matching the passed name too.